src/main/java/jep/Jep.java

```diff
@@ -497,26 +353,6 @@ public Object getValue(String str) throws JepException {
     private native Object getValue(long tstate, String str, Class<?> clazz)
             throws JepException;
 
-    /**
-     * @deprecated use Python 3 bytes object instead and
-     *             {@link #getValue(String,Class)} with byte[].class
-     *
-     *             Retrieves a Python string object as a Java byte[].
-     * 
-     * @param str
-     *            the name of the Python variable to get from the
-     *            sub-interpreter's global scope
-     * @return an <code>Object</code> array
-     * @throws JepException
-     *             if an error occurs
-     */
-    @Deprecated
-    public byte[] getValue_bytearray(String str) throws JepException {
-        isValidThread();
-
-        return getValue_bytearray(this.tstate, str);
-    }
-
     private native byte[] getValue_bytearray(long tstate, String str)
```

Comment:
Should remove the private methods when they aren't used anymore

Reply:
Fixed